make/addons.mk

```diff
@@ -7,9 +7,9 @@ export NODE_FEATURE_DISCOVERY_VERSION := 0.15.2
 export CLUSTER_AUTOSCALER_VERSION := 9.37.0
 export AWS_CSI_SNAPSHOT_CONTROLLER_VERSION := v6.3.3
 export AWS_EBS_CSI_CHART_VERSION := v2.28.1
-export NUTANIX_STORAGE_CSI_CHART_VERSION := v3.0.0-beta.1912
-export NUTANIX_SNAPSHOT_CSI_CHART_VERSION := v6.3.2
-export LOCAL_PATH_CSI_CHART_VERSION := v0.0.29
+export NUTANIX_STORAGE_CSI_CHART_VERSION := 3.0.0-beta.1912
+export NUTANIX_SNAPSHOT_CSI_CHART_VERSION := 6.3.2
+export LOCAL_PATH_CSI_CHART_VERSION := 0.0.29
```

Comment:
Any way we can avoid changing these since the upstream versions do have the "v"?

Reply:
ill see if we can - mindthegap was being fussy about these previously

Reply:
Mindthegap fussy? Never!

Reply:
i looked into this and its not the fault of mindthegap. for example

with this repos.yaml

```yaml
repositories:
  metallb:
    repoURL: https://metallb.github.io/metallb
    charts:
      metallb:
      - v0.14.5
```

mindthegap is able to find it correctly and create the oci archive. 

but since the index.yaml for metallb does not have the v 

```yaml 
apiVersion: v1
entries:
  metallb:
  - apiVersion: v2
    appVersion: v0.14.5
    created: "2024-04-19T14:44:27.58227863Z"
    dependencies:
    - condition: crds.enabled
      name: crds
      repository: ""
      version: 0.14.5
    - condition: frrk8s.enabled
      name: frr-k8s
      repository: https://metallb.github.io/frr-k8s
      version: 0.0.11
    description: A network load-balancer implementation for Kubernetes using standard
      routing protocols
    digest: 12e8d6a6ef2bf0d6450c10fba55c94282ef2d3559f78d2fbf626000d13baa9ef
    home: https://metallb.universe.tf
    icon: https://metallb.universe.tf/images/logo/metallb-white.png
    kubeVersion: '>= 1.19.0-0'
    name: metallb
    sources:
    - https://github.com/metallb/metallb
    type: application
    urls:
    - https://github.com/metallb/metallb/releases/download/metallb-chart-0.14.5/metallb-0.14.5.tgz
    version: 0.14.5
```

we don't see that when we list the tags on the chart 

```
$ curl -k https://localhost:5000/v2/charts/metallb/tags/list | jq .
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100    44  100    44    0     0   7604      0 --:--:-- --:--:-- --:--:--  8800
{
  "name": "charts/metallb",
  "tags": [
    "0.14.5"
  ]
}

``` 

with all of this being said - i think stripping the v from these makes sense because we need to be able to access this as its served through the helm registry